backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -20,12 +20,22 @@
     @Autowired
     private PersonService ps;
 
-    public List<Person> getPatients(UUID facilityId) {
-        return ps.getPatients(facilityId);
+    public List<Person> getPatients(UUID facilityId, long pageNumber, long pageSize, boolean showDeleted) {
+        if (pageNumber < 0) {
+            pageNumber = PersonService.DEFAULT_PAGINATION_PAGEOFFSET;
+        }
+        if (pageSize < 1) {
+            pageSize = PersonService.DEFAULT_PAGINATION_PAGESIZE;
+        }
+        if (facilityId == null) {
+            return showDeleted ? ps.getAllPatientsInclDeleted() : ps.getAllPatients();
+        }
+        return showDeleted ? ps.getPatients(facilityId) : ps.getPatientsInclDeleted(facilityId);
```

Comment:
I would expect this to be
```
         return showDeleted ? ps.getArchivedPatients(facilityId) : ps.getPatients(facilityId); 
```

Additionally, I think there is a bug here. Seems that `getPatients` and `getPatientsInclDeleted` should be swapped

Reply:
Good catch on them being swapped. My unit tests should have caught this case, so I'll double check it there, too.